dev/gphl/Debian9_qt4/start_mxcube2

```diff
@@ -7,12 +7,12 @@ CONTAINER=mx3_deb9_qt4_gphl2
 # MXCuBE additional lookup directories
 #
 # Examples of alternative path settings:
-HO_SERVER="/MXCuBE/mxcube/ExampleFiles/HardwareObjects.xml"
+HO_SERVER="/MXCuBE/mxcube/HardwareRepository/"
 # HARDWARE_REPOSITORY_SERVER="/MXCuBE/mxcube/ExampleFiles/HardwareObjects.xml:/MXCuBE/mxcube/repo_test_1"
 # CUSTOM_HARDWARE_OBJECTS_PATH="/MXCuBE/mxcube/extra_HO_test_1:/MXCuBE/mxcube/extra_HO_test_2"
 # CUSTOM_BRICKS_PATH="/MXCuBE/mxcube/extra_brick_test_1:/MXCuBE/mxcube/extra_brick_test_2"
-HARDWARE_REPOSITORY_SERVER="${HO_SERVER}/gphl_mock:/${HO_SERVER}:/${HO_SERVER}/embl_hh_p14"
-CUSTOM_HARDWARE_OBJECTS_PATH=""
+HARDWARE_REPOSITORY_SERVER="${HO_SERVER}/configuration/gphl_mock:/${HO_SERVER}/test/xml-qt:${HO_SERVER}/configuration/alba_xaloc13"
```

Comment:
This path is ALBA specific, is that the intention ?

Reply:
Sort of, yes. This is inside the 'gphl/' directory, which makes it global-phasing specific. Since I test relative to different beamlines  I need to have varying specific directories on the lookup path, and it seemed easier to have all the configuring in one place, and checked in, and then change it at need. That way it is visible how tings work, in a way it would not be if I relied on some other mechanism, like local environment variables. But if you feel it is not a good idea to have specific locations checked in, even in a gphl/ directory, I can change to a different model